source/parser.hera

```diff
@@ -381,14 +381,26 @@ ClassHeritage
   ImplementsClause
 
 ExtendsClause
-  __ ExtendsToken __ ExtendsTarget
+  ExtendsToken __ ExtendsTarget
 
 ExtendsToken
-  # NOTE: Added "<" extends shorthand
-  "<" ->
-    return { $loc, token: "extends" }
-  "extends" NonIdContinue ->
-    return { $loc, token: $1 }
+  # NOTE: Added "<" extends shorthand which must be surrounded by spaces
```

Comment:
This comment feels off. The shorthand *doesn't* need to be surrounded by spaces (as the tests show), but the expansion does. Otherwise looks great!

Reply:
You're right, that was from an intermediate version. I've updated the comment.